test/scripts/test_imgtestlib.py

```diff
@@ -21,3 +33,49 @@ def test_read_seed():
     # check that it's read without error - no need to test the value itself
     seed_env = rng_seed_env()
     assert "OSBUILD_TESTING_RNG_SEED" in seed_env
+
+
+test_container = "registry.gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/manifest-list-test"
+
+# manifest IDs for
+#  registry.gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/manifest-list-test:latest
+manifest_ids = {
+    "amd64": "sha256:601c98c8148720ec5c29b8e854a1d5d88faddbc443eca12920d76cf993d7290e",
+    "arm64": "sha256:1a19a94647b1379fed8c23eb7553327cb604ba546eb93f9f6c1e6d11911c8beb",
+}
+
+# image IDs for
+#  registry.gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/manifest-list-test:latest
+image_ids = {
+    "amd64": "sha256:dbb63178dc9157068107961f11397df3fb62c02fa64f697d571bf84aad71cb99",
+    "arm64": "sha256:62d2a7b3bf9e0b4f3aba22553d6971227b5a39f7f408d46347b1ee74eb97cb20",
+}
+
+
+@pytest.mark.parametrize("arch", ["amd64", "arm64"])
```

Comment:
(nipitck) given that `["amd64", "arm64"]` is repeated maybe `TEST_ARCHES = ["amd64", "arm64"]` or something? Then if we add s390x or riscv64 we only need to change one place?